offline_migration/upgrade_glib2_pango32bit.pm

```diff
@@ -0,0 +1,26 @@
+# SUSE's openQA tests
+#
+# Copyright (C) 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use strict;
+use base "consoletest";
+use testapi;
+
+sub run() {
+    my $repo_url = 'http://download.suse.de/ibs/SUSE:/SLE-11-SP1:/Update/standard/';
+    select_console 'root-console';
+    script_run "chmod 444 /usr/sbin/packagekitd";    # packagekitd will be not executable
+    script_run "pkill -f packagekitd";
+    script_run "zypper ar $repo_url repo";
```

Comment:
The code was just copied from #1547 which only to temperately use. if you like zypper_call, i will fix that :)